src/fragments/lib/push-notifications/react-native/migrate_from_previous_version/migrate-from-previous-version.mdx

```diff
@@ -0,0 +1,142 @@
+
+```bash
+npm uninstall @aws-amplify/pushnotification
+```
+
+2. Add the new native module to your project.
+
+```bash
+npm install @aws-amplify/rtn-push-notification
+```
+
+## Update build configurations
+
+<BlockSwitcher>
+<Block name="iOS">
+
+### Update your `Podfile`
+
+Amplify Push Notification now requires a minimum target of iOS 13.0.
+
+1. Open `Podfile` located inside the `/ios` folder of your React Native project with a text editor.
+
+2. Update `platform :ios` to at least `13.0`
+
+```ruby
+platform :ios, '13.0'
+```
+
+3. Install the new pods by running the following command at the root of your React Native project.
+
+```bash
+npx pod-install
+```
+</Block>
+<Block name="Android">
+
+### Update gradle scripts
+
+Amplify Push Notification now requires a minimum target of Android SDK API level 24.
+
+1. Open `build.gradle` located inside the `/android` folder of your React Native project with a text editor.
+
+2. Update the `minSdkVersion` value in the `buildscript` block to at least `24`.
+
+```groovy
+buildscript {
+  ext {
+    ...
+    minSdkVersion = 24
+    ...
+  }
+}
+```
+
+3. Open the application `build.gradle` located inside the `/android/app` folder of your React Native project with a text editor.
+
+4. Remove firebase dependencies from the `dependencies` block.
+
+```diff
+dependencies {
+  ...
+-   implementation "com.google.firebase:firebase-core:15.0.2"
+-   implementation "com.google.firebase:firebase-messaging:15.0.2"
+  ...
+}
+```
+
+### Update the manifest
+
+5. Open `AndroidManifest.xml` located inside the `/android/app/src/main` folder.
+
+6. Remove the old service block from the `application`.
+
+```diff
+<application ... >
+  ...
+- <!--[START Push notification config -->
+- <service
+-   android:name="com.amazonaws.amplify.pushnotification.RNPushNotificationMessagingService"
+-   android:exported="false">
+-   <intent-filter>
+-     <action android:name="com.google.firebase.MESSAGING_EVENT"/>
+-   </intent-filter>
+- </service>
+- <!-- [END Push notification config -->
+  ...
+</application>
+```
+
+</Block>
+</BlockSwitcher>
+
+## Update your application code
+
+### Add required polyfills
+
+You need to add the crypto.getRandomValues and URL polyfill to your application's entry point file (in most React Native apps this will be the top level `index.js`).
+
+```js
+// Example index.js
+import 'react-native-get-random-values';
+import 'react-native-url-polyfill/auto';
+...
+```
+
+### Remove Analytics (Optional)
+
+**If you do not** use Amplify Analytics in your application but have previously imported it in your code just to implement Amplify Push Notification, you should be able to remove it now. Analytics is no longer a required to be configured in your frontend application to use Push Notification.
+
+### Replace APIs
+
+Please refer to the [Getting started](/lib/push-notifications/getting-started) portion of this guide to start learning more about the new Amplify Push Notifications.
```

Comment:
Real question here, are we treating `Amplify Push Notification(s)` as a proper noun and as a single entity? Everywhere else on this page it is `Amplify Push Notification` but here it is plural. Notifications I think could be plural in other context but here I think we are basically saying "...about the new <category>" 

I also could be thinking about it too much. Are they interchangeable because the main category header is "Push Notifications"